tests/components/modbus/test_sensor.py

```diff
@@ -15,6 +15,7 @@
     CONF_MIN_VALUE,
     CONF_NAN_VALUE,
     CONF_PRECISION,
+    CONF_REGISTER_SIZE_BYTES,
```

Comment:
The mixing of "normal" testing and "Non-standard" testing is making it very difficult to see if the new test cases, test everything (including virtual).

Please do not change this file, apart from 1 test, with "register_size: 2", just to show the normal works.

Please make a new file e.g. test_sensor_sizes.py, and have all the new tests there, that way it is easy to compare.